university_compiler.includes

```diff
@@ -0,0 +1,8 @@
+source/include
```

Comment:
As this file has nothing to do with project, I think you should delete these file. As well, you should consider deleting following files: `university_compiler.files`, `university_compiler.config` and `university_compiler.creator`.